pkgs/data/fonts/noto-fonts/default.nix

```diff
@@ -171,8 +171,10 @@ in
     enableParallelBuilding = true;
 
     installPhase = ''
+      runHook preInstall
       mkdir -p $out/share/fonts/noto
-      cp NotoColorEmoji.ttf fonts/NotoEmoji-Regular.ttf $out/share/fonts/noto
```

Comment:
This seems right. There is a new black-and-white emoji font, discussed in https://github.com/googlefonts/noto-emoji/issues/390, but if that gets packaged it should probably be a separate package.

Reply:
Good to know! Would be nice to be able to build that from source then, as well.